treeherder/webapp/api/bugzilla.py

```diff
@@ -0,0 +1,42 @@
+import requests
+from django.conf import settings
+from rest_framework import (status,
+                            viewsets)
+from rest_framework.decorators import list_route
+from rest_framework.response import Response
+
+from treeherder.etl.common import make_request
+
+
+class BugzillaViewSet(viewsets.ViewSet):
+    permission_classes = (IsAuthenticatedOrReadOnly,)
+
+    @list_route(methods=['post'])
+    def create_bug(self, request):
+        """
+        Create a bugzilla bug with passed params
+        """
+
+        params = request.data
+        url = settings.BZ_API_URL + "/rest/bug"
+        headers = {
+            'X_BUGZILLA_API_KEY': settings.BZ_API_KEY
+        }
+        data = {
+            'product': params["product"],
+            'component': params["component"],
+            'summary': params["summary"],
+            'keywords': params["keywords"],
+            'version': params["version"],
+            'description': params["description"],
+            'comment_tags': "treeherder",
+            # XXX Should implement dependson, blocks, needinfo, and ccstring fields
+        }
+
+        try:
+            response = make_request(url, method='POST', headers=headers, json=data)
+        except requests.exceptions.HTTPError as e:
+            response = e.response
+            return Response({"failure": response.json(), "headers": headers, "settings": settings.BUGZILLA_API_KEY}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
On failures this sends the secret API key back to the UI, which is probably not what we want :-)

There are also two cases that need to be handled:
* An HTTPError where the Bugzilla API gave as a json response (eg we sent an invalid payload and it replies with an error message in json format)
* An HTTPError where the content is not decodable json (eg the Bugzilla API is down and the load balancer gives us a 500 and a text or html response).

At the moment this will cause exceptions in the second case, due to the `.json()`.

I'd put the `.json()` in a try-except and if it succeeds return the value of the `message` key as part of the error message.